components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -56,6 +57,28 @@
   return [UIColor lightGrayColor];
 }
 
+@implementation MDCTextInputClearButton
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+  CGFloat width = CGRectGetWidth(self.bounds);
+  CGFloat height = CGRectGetHeight(self.bounds);
+  CGFloat verticalInset = MIN(0, -(MDCTextInputClearButtonTouchTargetSize - height) / 2);
+  CGFloat horizontalInset = MIN(0, -(MDCTextInputClearButtonTouchTargetSize - width) / 2);
+  self.minimumTouchTargetInsets =
+      UIEdgeInsetsMake(verticalInset, horizontalInset, verticalInset, horizontalInset);
+}
+
+- (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event {
+  if (!UIEdgeInsetsEqualToEdgeInsets(self.minimumTouchTargetInsets, UIEdgeInsetsZero)) {
+    return CGRectContainsPoint(
+        UIEdgeInsetsInsetRect(CGRectStandardize(self.bounds), self.minimumTouchTargetInsets),
+        point);
+  }
+  return [super pointInside:point withEvent:event];
+}
```

Comment:
Note: a healthy follow-up to this change may be to move this pointInside logic into a parent view so that this behavior is no longer a "magic" aspect of the clear button.